{"name": "test_login01[case2]", "status": "broken", "statusDetails": {"message": "NameError: name 'act_res' is not defined", "trace": "self = <testcase.test_login.TestLogin object at 0x00000256600C8CD0>\ncase = <comon.excel_util.CaseData object at 0x000002566007B610>\n\n    @pytest.mark.parametrize('case',ExcelUtil().read_excel('login'))\n    def test_login01(self, case):\n        print(case)\n        allure.attach(body=case.url, name='接口路径')\n        allure.attach(body=case.case_data, name='请求参数')\n        allure.attach(body=case.res, name='预期结果')\n        logger.info(\"*************** 开始执行用例 ***************\")\n        # logger.info('----------->开始执行测试案例:{}'.format(case['case_title']))\n        response = requests.get(url=case.url, data=eval(case.case_data))\n        # act_res=response.json()\n>       print(act_res)\nE       NameError: name 'act_res' is not defined\n\ntestcase\\test_login.py:26: NameError"}, "attachments": [{"name": "接口路径", "source": "ecdc9ef9-ccf1-49be-adf3-bd8a8b5cbe57-attachment.attach"}, {"name": "请求参数", "source": "58ec8e52-2e26-4133-848d-ec8aa24b61c5-attachment.attach"}, {"name": "预期结果", "source": "fe1d0d19-5de8-41b1-a240-b6893b29f1a8-attachment.attach"}], "parameters": [{"name": "case", "value": "<comon.excel_util.CaseData object at 0x000002566007B610>"}], "start": 1650292603209, "stop": 1650292603216, "uuid": "ba1bd01c-e68b-4de5-9bba-e8d7afdacb10", "historyId": "f1448aed770d7fdd91df4365d7b3461a", "testCaseId": "f0a3740a932ee8262933e5b94534ef30", "fullName": "testcase.test_login.TestLogin#test_login01", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Mikey"}, {"name": "thread", "value": "15044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}